### PR TITLE
Issue 60 : contao content override

### PR DESCRIPTION
**Pull request by ChipsVII**

### CODE REVIEW

File: src/Resources/contao/config/config.php [hunk 1]
```diff
@@ -67,6 +67,8 @@
     unset($GLOBALS['BE_MOD']['design']['i18nl10n']);
 }
 
+// ComponentStyleSelect override
+$GLOBALS['BE_FFL']['stylemanager'] = 'WEM\\SmartgearBundle\\Widgets\\ComponentStyleSelect';
```
Comment (LupusVII) [inline src/Resources/contao/config/config.php:71]: Try to use `WEM\SmartgearBundle\Widgets\ComponentStyleSelect::class` instead

Comment (ChipsVII) [reply src/Resources/contao/config/config.php]: done, it works

File: src/Backend/Module/Core/ConfigurationStep/Website.php [hunk 2]
```diff
@@ -316,20 +324,58 @@ protected function createUserGroups(): array
...
+            'rsce_testimonials',
+        ]);
         $objUserGroup->save();
         $userGroups['administrators'] = $objUserGroup;
 
         $objUserGroup = UserGroupModel::findOneByName($GLOBALS['TL_LANG']['WEMSG']['INSTALL']['WEBSITE']['UsergroupRedactorsName']) ?? new UserGroupModel();
         $objUserGroup->tstamp = time();
         $objUserGroup->name = $GLOBALS['TL_LANG']['WEMSG']['INSTALL']['WEBSITE']['UsergroupRedactorsName'];
-        $objUserGroup->modules = 'a:8:{i:0;s:7:"article";i:1;s:5:"files";}';
+        $objUserGroup->modules = 'a:2:{i:0;s:4:"article";i:1;s:5:"files";}';
```
[the remark this answers is not in the document]
Comment (ChipsVII) [reply src/Backend/Module/Core/ConfigurationStep/Website.php]: done

File: src/Backend/Module/Core/ConfigurationStep/Website.php [hunk 3]
```diff
@@ -316,20 +324,58 @@ protected function createUserGroups(): array
...
+        // $objUserGroup->modules = 'a:8:{i:0;s:4:"page";i:1;s:7:"article";i:2;s:4:"form";i:3;s:5:"files";i:4;s:16:"nc_notifications";i:5;s:4:"user";i:6;s:3:"log";i:7;s:11:"maintenance";}';
+        $objUserGroup->modules = serialize(['page', 'article', 'form', 'files', 'nc_notifications', 'user', 'log', 'maintenance']);
         // $objUserGroup->pagemounts = '';
         // $objUserGroup->alpty = 'a:3:{i:0;s:7:"regular";i:1;s:7:"forward";i:2;s:8:"redirect";}';
         // $objUserGroup->filemounts = 'a:1:{i:0;s:16:"'.$objMediaFolder->getModel()->uuid.'";}';
         // $objUserGroup->fop = 'a:4:{i:0;s:2:"f1";i:1;s:2:"f2";i:2;s:2:"f3";i:3;s:2:"f4";}';
         // $objUserGroup->imageSizes = 'a:3:{i:0;s:12:"proportional";i:1;s:3:"box";i:2;s:4:"crop";}';
-        // $objUserGroup->alexf = Util::addPermissions($this->getCorePermissions());
+        $objUserGroup->alexf = Util::addPermissions($this->getCorePermissions());
+        $objUserGroup->elements = serialize([
```
[the remark this answers is not in the document]
Comment (ChipsVII) [reply src/Backend/Module/Core/ConfigurationStep/Website.php]: Made issue #68 for this.

**Pull request merged**